docs/framework/20categories/10network/network.md

```diff
@@ -1,2 +1,18 @@
 ## Network Consensus Risk
-TODO: The EEA Security Guidelines has some good words for this section.
+Cross-chain protocols enable the coordination of state changes across networks. This typically requires sending state information from a source network to one or more destination networks and performing consequent state changes. A fundamental assumption in this process is that: a) states in the source network are valid and final according to its consensus rules, and b) the underlying network can process all transactions in a timely manner.
+
+From the perspective of risk in cross-chain protocols, there are at least three considerations relating to these assumptions:
+1. Protocols should only communicate states that have been finalized in the source network
+1. Protocol security should be resilient to network liveness failures
+1. Protocols should mitigate the impact of network safety violations
+
+#### Transaction Finality
+Finality models vary significantly across networks. They could be _probabilistic_ or _deterministic_, _instant_ or _eventual_. Networks with _deterministic_ finality offer absolute assurance about the state of a network after some time. In contrast, probabilistic models offer only degrees of confidence, with more crypto-economic assurance achieved over time. For _deterministic_ models, finality could be instantly achieved after a single block or eventually after several blocks. Networks with slower finality times increase latency in cross-chain communication, which impacts the customer experience. However, cross-chain protocols must ensure that the finality of a given state is achieved before relaying the state across chains. Cross-chain protocols, which optimize for performance and relay information across chains without sufficiently waiting for finality, risk creating inconsistencies across networks and the possible loss of user funds.
+
+#### Network Liveness Failures
+Some cross-chain protocols rely considerably on the underlying networks processing transactions within a given period. For instance, protocols might assume cross-chain messages or fraud proofs will be submitted within a period. Because some consensus protocols explicitly trade off liveness for safety under certain conditions, cross-chain protocols must account for these conditions.
```

Comment:
trade-off

Reply:
I think adding a sentence at the end of this paragraph being explicit about what the trade-off is. For instance, start the sentence saying, "That is, ..."

Reply:
makes sense. added sentence to provide more clarity.